var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -180,10 +180,13 @@ def yes_no(varstr):
             options += ['MPICXX=%s' % spec['mpi'].mpicxx]
 
         if '+hypre' in spec:
-            options += [
-                'HYPRE_DIR=%s' % spec['hypre'].prefix,
-                'HYPRE_OPT=-I%s' % spec['hypre'].prefix.include,
-                'HYPRE_LIB=-L%s' % spec['hypre'].prefix.lib + ' -lHYPRE']
+            hypre = spec['hypre']
+            hp = hypre.prefix
+            options += ['HYPRE_DIR=%s' % hp,
+                        'HYPRE_OPT=-I%s' % hp.include,
+                        'HYPRE_LIB=-L%s' % (hp.lib + ' -lHYPRE ' + ' '.join(
+                            [hypre['lapack'].libs.ld_flags,
```

Comment:
if you first add `lapack` and `blas` and then call `ld_flags` then you won't have duplicates (see line 192 below)